src/account/azext_account/azext_metadata.json

```diff
@@ -1,4 +1,3 @@
 {
-    "azext.isExperimental": true,
```

Comment:
We need to confirm whether we only need GA a command group `az account alias` or GA the whole extension

Reply:
For the [scenario I have in mind](https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/programmatically-create-subscription-enterprise-agreement?tabs=azure-cli), az account alias is sufficient.

Reply:
OK, thanks~